build/vsts_build.yaml

```diff
@@ -83,6 +83,7 @@ phases:
   - task: MicroBuildSigningPlugin@1
     inputs:
       signType: "$(SigningType)"
+      esrpSigning: "true"
```

Comment:
how does this work for test signing?

Reply:
this is ignored for test sign and only matter when it's real sign.

Reply:
And are the packages signed in place? Or are they copied to some location?